fast64_internal/f3d/f3d_material.py

```diff
@@ -2460,45 +2460,54 @@ class RDPSettings(bpy.types.PropertyGroup):
         name="Z Buffer",
         default=True,
         update=update_node_values_with_preset,
+        description="Turns on/off Z-Buffer.  Z-Buffer set to 0 if disabled."
```

Comment:
This is not correct. Not only does this not set the Z buffer itself to zero as is claimed, but it does not set the Z screen coordinates of the triangle to zero--they end up being garbage data in the RDP. What this does is selects whether Z coordinates are computed on the RSP and sent to the RDP or not; it skips sending them if not. I'd suggest something like "Compute and send Z coordinates of tris. Disable if not reading or writing the Z buffer in the blender."